packages/flutter/README.md

```diff
@@ -7,110 +7,3 @@ focused on low-latency input and high frame rates on Android and iOS.
-```
-
-The example that follows can be viewed as a template for using Flutter's localizations classes. It defines a single class, MyLocalizations, that contains the app's localized resources (just one for the example). It uses the initializeMessages() function generated by the Dart intl package to load resources and Intl.message() to look them up.
-
-```dart
-class MyLocalizations {
-  MyLocalizations(this.locale);
-
-  final Locale locale;
-
-  static Future<MyLocalizations> load(Locale locale) {
-    return initializeMessages(locale.toString())
-      .then((Null _) {
-        return new MyLocalizations(locale);
-      });
-  }
-
-  static MyLocalizations of(BuildContext context) {
-    return Localizations.of<MyLocalizations>(context, MyLocalizations);
-  }
-
-  String title() => Intl.message('<title>', name: 'title', locale: locale.toString());
-  // ... more Intl.message() methods like title()
-}
-```
-
-A class based on the `intl` package imports a generated message catalog that provides
-the `initializeMessages()` function and the per-locale backing store for `Intl.message()`.
-The message catalog is produced by an `intl` tool that analyzes the source code for
-classes that contain `Intl.message()` calls. In this case that would just be the
-`MyLocalizations` class.
-
-An instance of `MyLocalizations` is created by a simple LocalizationsDelegate:
-
-```dart
-class MyLocalizationsDelegate extends LocalizationsDelegate<MyStrings> {
-  @override
-  Future<MyStrings> load(Locale locale) => MyStrings.load(locale);
-
-  @override
-  bool shouldReload(MyLocalizationsDelegate old) => false;
-}
-```
-
-An instance of `MyLocalizationsDelegate` is provided to application's MaterialApp (or its WidgetsApp for non-material applications):
-
-```dart
-class MyApp extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return new MaterialApp(
-      localizationsDelegates: <LocalizationsDelegate>[
-        new MyLocalizationsDelegate()
-      ],
-      supportedLocales: <Locale>[
-        const Locale('en', 'US'), // US English
-        const Locale('fr', 'CA'), // Candian French
-      ],
-      child: home: new Scaffold(
-        appBar: new AppBar(
-          title: new Text(MyLocalizations.of(context).title),
-          ...
-        ),
-      ),
-    );
-  }
-}
-```
-
-As you can see, the app's `AppBar` title comes from an instance of `MyLocalizations` produced by `MyLocalizations.load()` via `MyLocalizationsDelegate`. If the device's locale changes, then the app will be rebuilt with a new `MyLocalizations` object, one constructed for the new locale. Any reference to `MyLocalizations.of()` anywhere in the app will causes its context to be rebuilt if the locale changes.
-
-The [MaterialApp] `supportedLocales` parameter limits locale changes. In this case the app will only accept the US English or French Canadian locales and it will substitute US English (the first locale in the list) for anything else.
```

Comment:
"the first locale" --> "or the first locale"
To make it clear that US English is not required to be the first locale.

Reply:
Done